compiler/qsc/src/interpret/stateful.rs

```diff
@@ -154,23 +125,27 @@ impl<'a> NodeLookup for Lookup<'a> {
 }
 
 pub struct Interpreter {
```

Comment:
Another cosmetic change: I rearranged and roughly grouped the fields here because they were getting real hard to follow. In the future, I hope to separate out the "compilation state" group of fields to a different struct that encapsulate the incremental compilation bits.